turbinia/workers/__init__.py

```diff
@@ -22,13 +22,43 @@
 import os
 import platform
 import time
+import traceback
 import uuid
 
 from turbinia import config
 from turbinia import output_writers
 from turbinia import TurbiniaException
 
 
+# pylint: disable=unused-argument
+def results_handler(func, *args, **kwargs):
```

Comment:
Seeing that this decorator is needed by all subclasses that overrides run() could we do this with a hook instead? E.g, in TurbiniaTask:

```
TurbiniaTask(object):
  def run() 
   # do stuff here, like in the decorator
    self.do_run()

   def do_run():
     return NotImplemented

```
Then just implement do_run in all the classes that subclasses this.
WDYT?

Reply:
Yeah, that's a good idea.  I'll refactor it.